fix/gsi/glmwest_full_2km_lat_lon.nc

```diff
@@ -0,0 +1 @@
+../.agent/gsi/glmwest_full_2km_lat_lon.nc
```

Comment:
The 3 glm* fix files are available on Jet/Hera. I also sync'ed them to Orion/Hercules.
@ShunLiu-NOAA could you check whether they are available on wcoss2? If not, they can be copied from one of the above platforms.

Reply:
@guoqing-noaa I'll work to get them on wcoss2 if needed.

Reply:
@MatthewPyle-NOAA - got them onto cactus (in `/lfs/h2/emc/lam/noscrub/emc.lam/FIX_RRFS/gsi`), but need to remember to get them onto dogwood once that system returns.

Reply:
Thanks, @MatthewPyle-NOAA !

Reply:
The glm fixed files now are on the dogwood side of WCOSS as well.